cpp/src/IceDB/IceDB.cpp

```diff
@@ -35,7 +35,13 @@ LMDBException::ice_print(ostream& out) const
     out << ": " << mdb_strerror(_error);
 }
 
-#ifndef ICE_CPP11_MAPPING
+#ifdef ICE_CPP11_MAPPING
```

Comment:
Presumably we should remove all these ICE_CPP11_MAPPING since IceDB and the users of IceDB are all "new mapping" now.

Reply:
Fixed